From 586f70b854dec9ed0809c9dad63d18138b93fa53 Mon Sep 17 00:00:00 2001
From: Axel Naumann <Axel.Naumann@cern.ch>
Date: Mon, 26 Aug 2013 11:47:25 +0200
Subject: [PATCH 2/9] Add cling as tool to be built.

---
 .gitignore | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/.gitignore b/.gitignore
index 6a183cd..4bbdb50 100644
--- a/.gitignore
+++ b/.gitignore
@@ -50,6 +50,8 @@ runtimes/*
 !runtimes/*.*
 # Clang, which is tracked independently.
 tools/clang
+# Cling, which is tracked independently.
+tools/cling
 # LLDB, which is tracked independently.
 tools/lldb
 # lld, which is tracked independently.
-- 
2.6.0

