From 9839f4d83a98e33217ff30966e80f73b7e820912 Mon Sep 17 00:00:00 2001
From: Axel Naumann <Axel.Naumann@cern.ch>
Date: Thu, 9 Jun 2016 15:55:11 +0200
Subject: [PATCH 6/9] Mark weak symbols weak (apparently NFC).

---
 lib/ExecutionEngine/RuntimeDyld/RuntimeDyld.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/lib/ExecutionEngine/RuntimeDyld/RuntimeDyld.cpp b/lib/ExecutionEngine/RuntimeDyld/RuntimeDyld.cpp
index 06f9d51..5154c6a 100644
--- a/lib/ExecutionEngine/RuntimeDyld/RuntimeDyld.cpp
+++ b/lib/ExecutionEngine/RuntimeDyld/RuntimeDyld.cpp
@@ -621,7 +621,7 @@ void RuntimeDyldImpl::emitWeakSymbols(const ObjectFile &Obj,
     section_iterator SI = SymAndSection.second;
 
     uint32_t Flags = Sym.getFlags();
-    JITSymbolFlags RTDyldSymFlags = JITSymbolFlags::None;
+    JITSymbolFlags RTDyldSymFlags = JITSymbolFlags::Weak;
     if (Flags & SymbolRef::SF_Exported)
       RTDyldSymFlags |= JITSymbolFlags::Exported;
 
-- 
2.6.0

