From 834481c08e3c68f4472c8c9f412ee5b98f6b2e44 Mon Sep 17 00:00:00 2001
From: Axel Naumann <Axel.Naumann@cern.ch>
Date: Thu, 9 Jun 2016 15:55:11 +0200
Subject: [PATCH 07/10] Mark weak symbols weak (apparently NFC).

---
 lib/ExecutionEngine/RuntimeDyld/RuntimeDyld.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/lib/ExecutionEngine/RuntimeDyld/RuntimeDyld.cpp b/lib/ExecutionEngine/RuntimeDyld/RuntimeDyld.cpp
index 2df55cb..44d0b27 100644
--- a/lib/ExecutionEngine/RuntimeDyld/RuntimeDyld.cpp
+++ b/lib/ExecutionEngine/RuntimeDyld/RuntimeDyld.cpp
@@ -603,7 +603,7 @@ void RuntimeDyldImpl::emitWeakSymbols(const ObjectFile &Obj,
     section_iterator SI = SymAndSection.second;
 
     uint32_t Flags = Sym.getFlags();
-    JITSymbolFlags RTDyldSymFlags = JITSymbolFlags::None;
+    JITSymbolFlags RTDyldSymFlags = JITSymbolFlags::Weak;
     if (Flags & SymbolRef::SF_Exported)
       RTDyldSymFlags |= JITSymbolFlags::Exported;
 
-- 
2.7.4

