From 24109ef45b2a7702ce7632a6f4ea60ae772d5bbf Mon Sep 17 00:00:00 2001
From: Isuru Fernando <isuruf@gmail.com>
Date: Tue, 4 Aug 2020 21:06:30 -0500
Subject: [PATCH] pass through QEMU_LD_PREFIX & SDKROOT

---
 llvm/utils/lit/lit/TestingConfig.py | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/llvm/utils/lit/lit/TestingConfig.py b/llvm/utils/lit/lit/TestingConfig.py
index 55e2a764d8fa..2b333d33b3da 100644
--- a/llvm/utils/lit/lit/TestingConfig.py
+++ b/llvm/utils/lit/lit/TestingConfig.py
@@ -26,6 +26,8 @@ class TestingConfig(object):
             'LD_LIBRARY_PATH',
             'SYSTEMROOT',
             'TERM',
+            'QEMU_LD_PREFIX',
+            'SDKROOT',
             'CLANG',
             'LLDB',
             'LD_PRELOAD',
-- 
2.37.0.windows.1

