From 9233e2111c3c76d22ae5cf842e1f7e29e783d97e Mon Sep 17 00:00:00 2001
From: Isuru Fernando <isuruf@gmail.com>
Date: Tue, 4 Aug 2020 21:06:30 -0500
Subject: [PATCH 1/2] pass through QEMU_LD_PREFIX & SDKROOT

---
 utils/lit/lit/TestingConfig.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/utils/lit/lit/TestingConfig.py b/utils/lit/lit/TestingConfig.py
index d534d895e4ba..e3512471c156 100644
--- a/utils/lit/lit/TestingConfig.py
+++ b/utils/lit/lit/TestingConfig.py
@@ -21,7 +21,7 @@ class TestingConfig(object):
             'LLVM_DISABLE_CRASH_REPORT' : '1',
             }
 
-        pass_vars = ['LIBRARY_PATH', 'LD_LIBRARY_PATH', 'SYSTEMROOT', 'TERM',
+        pass_vars = ['LIBRARY_PATH', 'LD_LIBRARY_PATH', 'SYSTEMROOT', 'TERM', 'QEMU_LD_PREFIX', 'SDKROOT',
                      'CLANG', 'LLDB', 'LD_PRELOAD', 'ASAN_OPTIONS',
                      'UBSAN_OPTIONS', 'LSAN_OPTIONS', 'ADB', 'ANDROID_SERIAL',
                      'SSH_AUTH_SOCK', 'SANITIZER_IGNORE_CVE_2016_2143',
-- 
2.32.0.windows.2

