From 8409d6d58765ae9e6a9e68a771da1afcbe1690bc Mon Sep 17 00:00:00 2001
From: Isuru Fernando <isuruf@gmail.com>
Date: Tue, 4 Aug 2020 21:06:30 -0500
Subject: [PATCH 1/2] pass through QEMU_LD_PREFIX & SDKROOT

---
 llvm/utils/lit/lit/TestingConfig.py | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/llvm/utils/lit/lit/TestingConfig.py b/llvm/utils/lit/lit/TestingConfig.py
index 55e2a764d..2b333d33b 100644
--- a/llvm/utils/lit/lit/TestingConfig.py
+++ b/llvm/utils/lit/lit/TestingConfig.py
@@ -26,6 +26,8 @@ class TestingConfig(object):
             'LD_LIBRARY_PATH',
             'SYSTEMROOT',
             'TERM',
+            'QEMU_LD_PREFIX',
+            'SDKROOT',
             'CLANG',
             'LLDB',
             'LD_PRELOAD',
-- 
2.37.0 (Apple Git-136)

