From d7e9544b005c502ff443a8b5d5171225276a8391 Mon Sep 17 00:00:00 2001
From: Isuru Fernando <isuruf@gmail.com>
Date: Mon, 5 Apr 2021 11:32:11 -0500
Subject: [PATCH 2/2] Don't link libLLVM.dylib to llvm-config

---
 tools/llvm-config/CMakeLists.txt | 1 +
 1 file changed, 1 insertion(+)

diff --git a/tools/llvm-config/CMakeLists.txt b/tools/llvm-config/CMakeLists.txt
index 1b74355a4209..5df32b731b3e 100644
--- a/tools/llvm-config/CMakeLists.txt
+++ b/tools/llvm-config/CMakeLists.txt
@@ -5,6 +5,7 @@ set(BUILDVARIABLES_OBJPATH ${CMAKE_CURRENT_BINARY_DIR}/BuildVariables.inc)
 
 # Add the llvm-config tool.
 add_llvm_tool(llvm-config
+  DISABLE_LLVM_LINK_LLVM_DYLIB
   llvm-config.cpp
   )
 
-- 
2.32.0.windows.2

