From 76349353dd18e30a5e0c660cc848ae037134a704 Mon Sep 17 00:00:00 2001
From: Isuru Fernando <isuruf@gmail.com>
Date: Mon, 5 Apr 2021 11:32:11 -0500
Subject: [PATCH 2/2] Don't link libLLVM.dylib to llvm-config

---
 tools/llvm-config/CMakeLists.txt | 1 +
 1 file changed, 1 insertion(+)

diff --git a/tools/llvm-config/CMakeLists.txt b/tools/llvm-config/CMakeLists.txt
index aa5aa20257f9..9198ac389c79 100644
--- a/tools/llvm-config/CMakeLists.txt
+++ b/tools/llvm-config/CMakeLists.txt
@@ -5,6 +5,7 @@ set(BUILDVARIABLES_OBJPATH ${CMAKE_CURRENT_BINARY_DIR}/BuildVariables.inc)
 
 # Add the llvm-config tool.
 add_llvm_tool(llvm-config
+  DISABLE_LLVM_LINK_LLVM_DYLIB
   llvm-config.cpp
   # This utility doesn't use much of LLVM, so linking a shared library for the
   # entire thing is overkill. Avoiding that especially saves on build time when cross
-- 
2.32.0.windows.2

