Index: llvm/tools/llvm-config/CMakeLists.txt
===================================================================
--- llvm.orig/tools/llvm-config/CMakeLists.txt
+++ llvm/tools/llvm-config/CMakeLists.txt
@@ -5,6 +5,7 @@ set(BUILDVARIABLES_OBJPATH ${CMAKE_CURRE
 
 # Add the llvm-config tool.
 add_llvm_tool(llvm-config
+  DISABLE_LLVM_LINK_LLVM_DYLIB
   llvm-config.cpp
   # This utility doesn't use much of LLVM, so linking a shared library for the
   # entire thing is overkill. Avoiding that especially saves on build time when cross
