Index: llvm-project/llvm/tools/llvm-shlib/CMakeLists.txt
===================================================================
--- llvm-project.orig/llvm/tools/llvm-shlib/CMakeLists.txt	2023-10-31 15:32:13.827516450 -0500
+++ llvm-project/llvm/tools/llvm-shlib/CMakeLists.txt	2023-10-31 15:32:31.612069733 -0500
@@ -94,7 +94,7 @@
   set(LLVM_EXPORTED_SYMBOL_FILE ${LLVM_BINARY_DIR}/libllvm-c.exports)
 
   set(LIB_DIR ${LLVM_LIBRARY_DIR})
-  set(LIB_NAME ${LIB_DIR}/${CMAKE_SHARED_LIBRARY_PREFIX}LLVM)
+  set(LIB_NAME ${LIB_DIR}/${CMAKE_SHARED_LIBRARY_PREFIX}LLVM-${LLVM_VERSION_MAJOR}${LLVM_VERSION_SUFFIX})
   set(LIB_PATH ${LIB_NAME}${CMAKE_SHARED_LIBRARY_SUFFIX})
   set(LIB_EXPORTS_PATH ${LIB_NAME}.exports)
   list(APPEND LLVM_DYLIB_REQUIRED_EXPORTS ${LIB_EXPORTS_PATH})
